### ISSUE TITLE
LSeg-3D

### ISSUE THREAD

**Issue by HuitongJin** [CVMI-Lab/PLA#2]
Hello, your work is very good. In the paper, I saw that you implemented 3D semantic segmentation based on LSeg and used it as a baseline. Is it convenient to share the code of this part?

**Comment by Dingry** [comment]
Hi, you can create a new config inherited from https://github.com/CVMI-Lab/PLA/blob/main/tools/cfgs/scannet_models/spconv_clip_adamw.yaml and add base_class_idx, novel class_idx and ignore_class_idx just as in https://github.com/CVMI-Lab/PLA/blob/main/tools/cfgs/scannet_models/spconv_clip_base15_caption_adamw.yaml#L5-L7. Then it is the LSeg-3D baseline without our binary head or caption supervision.

**Comment by HuitongJin** [comment]
Hello, according to your suggestion, I am implementing LSeg-3D. By the way, does LSeg-3D use image information, such as color or depth, so as to drive the 3D backbone to learn better semantic information? Or is the image only used in the model you proposed?

**Comment by Dingry** [comment]
Yes, LSeg-3D does not utilize images but solely relies on 3D point clouds. We named it LSeg-3D because it follows the training pipeline of LSeg.

**Comment by HuitongJin** [comment]
Can you share the trained model parameters of LSeg-3D (B15/N4), I wanna to reproduce the results quickly.

**Comment by HuitongJin** [comment]
Both baseline and used binary header

**Comment by Dingry** [comment]
Hi, I believe implementing it on your own would be faster. Moreover, you only need to train LSeg-3D w/ binary decoder, as the binary decoder input is detached from the backbone. Testing it with MODEL.TASK_HEAD.CORRECT_SEG_PRED_BINARY set to False will yield the exact LSeg-3D prediction.

**Comment by HuitongJin** [comment]
Problem solved, thank you very much for your answer
